integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/PerformanceResource.java

```diff
@@ -1,50 +1,43 @@
 package com.redhat.service.smartevents.integration.tests.resources;
 
-import com.redhat.service.smartevents.integration.tests.common.Constants;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
-import io.restassured.response.Response;
+import software.tnb.hyperfoil.validation.HyperfoilValidation;
+import software.tnb.hyperfoil.validation.generated.model.Run;
 
 public class PerformanceResource {
 
     private static final String HYPERFOIL_URL = System.getProperty("performance.hyperfoil.url");
-    private static final String BASE_BENCHMARK_URL = HYPERFOIL_URL + "/benchmark/";
-    private static final String BASE_RUN_URL = HYPERFOIL_URL + "/run/";
-
-    public static void addBenchmark(String managerToken, String benchmarkContent, String contentType) {
-        createBenchmarkResponse(managerToken, benchmarkContent, contentType)
-                .then()
-                .log().ifValidationFails()
-                .statusCode(204);
+    private static final String BENCHMARK_CONTENT_FILE = "benchmark.hf.yaml";
+
+    // Creating validation object directly as TNB Hyperfoil default service starts/stops Hyperfoil, causing tests to fail on environment without Docker environment
+    public static HyperfoilValidation validation = new HyperfoilValidation(HYPERFOIL_URL);
```

Comment:
This time don't we need to add `performance.hyperfoil.url` into some yaml file?

Reply:
No.
The TNB is designed to start own Hyperfoil instance. Currently we run the Hyperfoil locally. For TNB to reach the locally running Hyperfoil we need to provide its URL.
Maybe in future when we will run it in OCP we can give the complete lifecycle control to TNB, removing the URL as it will be provided by TNB directly.

Reply:
TNB is not designed to leverage external Hyperfoil server by default.